### ISSUE TITLE
Reference on issue # 25

### ISSUE THREAD

**Issue by aikanarov** [trix-solutions/moncli#32]
Hi,

I have moncli 0.1.5 installed and have the same problem on (#25), I would like to change the status of an item but get this same error.

Version
`Name: moncli
Version: 0.1.5
Summary: Moncli, a pythonic/DDD client for Monday.com
Home-page: https://github.com/trix-solutions/moncli
Author: Andrew Shatz, Trix Solutions
Author-email: andrew.shatz@trix.solutions
License: UNKNOWN
Location: /home/ubuntu/anaconda3/lib/python3.6/site-packages
Requires: requests, nose
Required-by: `

Process:
`
--- Create Item
day = dt.datetime.now()
item_name = f'Getting Tweets | {day.strftime("%Y-%m-%d %H:%M:%S")}'
group_id='topics'

item = board.add_item(item_name=item_name, group_id=group_id)
print(f'Id: {item.id} | Name: {item.name}')

--- Try to change status

status_value = create_column_value(id='status', title='Status', column_type=ColumnType.status, label='Done')
item = item.change_column_value(column_value=status_value)

And get this error:

---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-270-35dcbe8fb828> in <module>
----> 1 status_value = create_column_value(id='status', title='Status', column_type=ColumnType.status, label='Done')
      2 item = item.change_column_value(column_value=status_value)

~/anaconda3/lib/python3.6/site-packages/moncli/columnvalue.py in create_column_value(id, column_type, title, **kwargs)
    680 
    681         if kwargs.__contains__('label'):
--> 682             return StatusValue(id, title, label=kwargs['label'], settings=kwargs['settings'])
    683 
    684         if kwargs.__contains__('index'):

KeyError: 'settings'

`

**Comment by Ashatz** [comment]
Hello @aikanarov,

The 'settings' that you see there refers to the column settings for the particular status column that you wish to create.   These settings must be pulled from the board's column object directly.  Unfortunately at this time, the _create_column_value_ method does not retrieve the column settings, as it is unable to determine which board to utilize.  

However, there is a workaround using the _board.get_column_value_ method.  You can do this by modifying your code in the following way.

`
status_value = board.get_column_value(id='status')
status_value.change_status_by_label('Done')
item = item.change_column_value(column_value=status_value)
`

Please let me know if this addresses your problem so that I may close this issue.  

Thanks!

**Comment by Ashatz** [comment]
@aikanarov, I have not heard back from you regarding this.  I will be closing this issue.  Please feel free to open another ticket should you run into any problems.  Thank you!
